package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@hyperplay/ui",
-  "version": "1.19.2",
+  "version": "1.19.3",
```

Maybe bump to a minor version since this is a improvement, not a fix.

```suggestion
  "version": "1.20.0",
```